development/docker-compose.yml

```diff
@@ -160,13 +161,15 @@ services:
       <<: *infrahub_config
       INFRAHUB_CONFIG: /source/development/infrahub.toml
       INFRAHUB_ADDRESS: http://server:8000
+      INFRAHUB_INTERNAL_ADDRESS: "http://server:8000"
```

Comment:
Not related to this PR but I'd like to get rid of this INTERNAL_ADDRESS param. It's redundant and we should use INFRAHUB_ADDRESS instead. The only reason for having INFRAHUB_INTERNAL_ADDRESS is if anyone wants to specify it within the infrahub.toml file, however with that file we don't support to add credentials so you have to use environment variables regardless. As we can see from this PR we are already running into problems by having settings for the same thing. :(

Reply:
I agree with that :+1: